dnf-behave-tests/fixtures/specs/dnf-ci-fedora-modular/postgresql-9.6.8-1.module_1710+b535a823.spec

```diff
@@ -26,20 +26,6 @@ PostgreSQL server, or on a remote machine that accesses a PostgreSQL server
 over a network connection.  The PostgreSQL server can be found in the
 postgresql-server sub-package.
 
-%package contrib
-Summary:        Extension modules distributed with PostgreSQL
-
-Provides:       postgresql-contrib = 9.6.8-1.module_1710+b535a823
-Provides:       postgresql-contrib(x86-64) = 9.6.8-1.module_1710+b535a823
-
-Requires:       rtld(GNU_HASH)
-Requires:       postgresql-libs(x86-64) = 9.6.8-1.module_1710+b535a823
-Requires:       postgresql(x86-64) = 9.6.8-1.module_1710+b535a823
-
-%description contrib
-The postgresql-contrib package contains various extension modules that are
-included in the PostgreSQL distribution.
-
 %package devel
```

Comment:
This subpackage should be also removed to stay consistent.
Also, all "Requires: postgresql-devel" should be removed from the postgresql-test package.